.travis.yml

```diff
@@ -14,6 +15,8 @@ script:
 - bundle exec rspec
 - bundle exec codeclimate-test-reporter
 matrix:
-  exclude:
+  allow_failures:
+    - rvm: ruby-head
     - rvm: 2.4.0
```

Comment:
Agreed - but as you can see this only excludes failures when Rails 3 and Ruby 2.4 are combined (because Rails 3 doesn.t run on Ruby 2.4).

Failures aren't allowed for Rails 4+ and any Ruby including 2.4.